### PR TITLE
chore: path analysis specify nodes

### PR DESCRIPTION
**Pull request by tyyzqmf**
----

*By submitting this pull request, I confirm that my contribution is made under the terms of the Apache-2.0 license*

## Summary

截屏2024-04-24 16 34 30

## Implementation highlights

(describe how the merge request does for feature changes, share the RFC link if it has)

## Test checklist

- [ ] add new test cases
- [ ] all code changes are covered by unit tests
- [ ] end-to-end tests
  - [ ] deploy web console with CloudFront + S3 + API gateway
  - [ ] deploy web console within VPC
  - [ ] deploy ingestion server
    - [ ] with MSK sink
    - [ ] with KDS sink
    - [ ] with S3 sink
  - [ ] deploy data processing
  - [ ] deploy data modeling
    - [ ] new Redshift Serverless
    - [ ] provisioned Redshift
    - [ ] Athena
  - [ ] deploy with reporting
  - [ ] streaming ingestion
    - [ ] with Redshift Serverless
    - [ ] with provisioned Redshift

## Is it a breaking change

- [ ] add parameters without default value in stack
- [ ] introduce new service permission in stack
- [ ] introduce new top level stack module

## Miscellaneous

- [ ] introduce new symbol link source file(s) to be shared among infra code, web console frontend, and web console backend

### CODE REVIEW

File: frontend/src/pages/analytics/path/AnalyticsPath.tsx [hunk 1]
```diff
@@ -715,64 +717,81 @@ const AnalyticsPath: React.FC<AnalyticsPathProps> = (
...
+                    'analytics:information.pathMergeNodeInfo'
+                  )}
+                />
+                <Toggle
+                  onChange={({ detail }) =>
+                    setMergeConsecutiveEvents(detail.checked)
+                  }
+                  checked={mergeConsecutiveEvents}
+                >
+                  {mergeConsecutiveEvents ? t('yes') : t('no')}
```
Comment (chenhaiyun) [inline frontend/src/pages/analytics/path/AnalyticsPath.tsx:793]: change to `ternary` from `utils.ts`

File: frontend/src/pages/analytics/path/AnalyticsPath.tsx [hunk 2]
```diff
@@ -715,64 +717,81 @@ const AnalyticsPath: React.FC<AnalyticsPathProps> = (
...
+                    enableChangeRelation={true}
+                    enableChangeMultiSelect={MultiSelectType.BASE}
                   />
-                  <Toggle
-                    onChange={({ detail }) =>
-                      setMergeConsecutiveEvents(detail.checked)
-                    }
-                    checked={mergeConsecutiveEvents}
-                  >
-                    {mergeConsecutiveEvents ? t('yes') : t('no')}
```
Comment (chenhaiyun) [inline frontend/src/pages/analytics/path/AnalyticsPath.tsx:773]: change to `ternary` from `utils.ts`

File: frontend/public/locales/zh-CN/analytics.json [hunk 3]
```diff
@@ -354,6 +354,8 @@
     "realtimeStarted": "实时流已启动",
     "analyticsStudioNotAvailable": "分析工作坊不可用",
     "setStartNode": "设置起始节点",
+    "showAllNodes": "显示所有节点",
+    "specifyNodes": "指定节点",
```
Comment (luorobin-a2z) [inline frontend/public/locales/zh-CN/analytics.json:358]: --> 显示指定节点

**Review by luorobin-a2z — APPROVED** [review]
with a few wording suggestion

**Review by zxkane — APPROVED** [review]

**Pull request merged**